packages/core/src/ceramic.ts

```diff
@@ -27,11 +34,11 @@ import { PathTrie, TrieNode, promiseTimeout } from './utils'
 
 import EthereumAnchorService from "./anchor/ethereum/ethereum-anchor-service"
 import InMemoryAnchorService from "./anchor/memory/in-memory-anchor-service"
-import { PinningBackendStatic } from "@ceramicnetwork/common"
-
 
 import { randomUint32 } from '@stablelib/random'
 
+const DEFAULT_BASE_DOC_CACHE_LIMIT = 500; // number of base docs stored in the cache
```

Comment:
Can we call this `DEFAULT_DOC_CACHE_LIMIT` (dropping the word "base") since it applies to both the base and commit cache?